profiles/seccomp/seccomp_default.go

```diff
@@ -160,6 +160,9 @@ func DefaultProfile() *types.Seccomp {
 				"ioprio_set",
 				"io_setup",
 				"io_submit",
+				"io_uring_setup",
+				"io_uring_enter",
+				"io_uring_register",
```

Comment:
Can you sort these alphabetically like the rest please

Reply:
done that